lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
Not following you. The other 3 places are just calls to `maxWidth(something)` which all happen within the public `.help()` method. I'm suggesting we stick with the one call here (line 137) and then pass `theWrap` as a 2nd argument to `maxWidth(something, theWrap)` for the other 3 (lines 156, 230, and 251), which would remove the need to call `getWrap()` again within the `maxWidth` function itself.

Think that will make the code too ugly or brittle?